### ISSUE TITLE
 Unable to load Maven meta-data from https://jcenter.bintray.com/com/applovin/applovin-sdk/maven-metadata.xml.

### ISSUE THREAD

**Issue by alexfernandojs** [ioridev/flutter_applovin_max#34]
ERROR Compiler plugin Repository off jcenter .
 > Could not resolve com.applovin:applovin-sdk:+.
     Required by:
         project :app
         project :app > project :flutter_applovin_max
      > Failed to list versions for com.applovin:applovin-sdk.
         > Unable to load Maven meta-data from https://jcenter.bintray.com/com/applovin/applovin-sdk/maven-metadata.xml.
            > Could not HEAD 'https://jcenter.bintray.com/com/applovin/applovin-sdk/maven-metadata.xml'. Received status code 502 from server: Bad Gateway

**Comment by landersonalmeida** [comment]
@alexfernandojs Hi Alex! how are u doing? i had the same issue, i don't know why that was happening but i forked the repository and changed the build.gradle file with the following contents:

`group 'dev.iori.flutter_applovin_max'
version '1.0'

buildscript {
    repositories {
        google()
        mavenCentral()
    }

    dependencies {
        classpath 'com.android.tools.build:gradle:3.5.0'
    }
}

rootProject.allprojects {
    repositories {
        google()
        mavenCentral()

    }
}

apply plugin: 'com.android.library'

android {
    compileSdkVersion 29

    defaultConfig {
        minSdkVersion 16
    }
    lintOptions {
        disable 'InvalidPackage'
    }
}

dependencies {
    implementation 'com.applovin:applovin-sdk:11.0.0'
}`

**Comment by alexfernandojs** [comment]
did this work for you?? . notice practically nothing has changed. because error and that the repository is offline. developer has move binaries to marven

**Comment by landersonalmeida** [comment]
> did this work for you?? . notice practically nothing has changed. because error and that the repository is offline. developer has move binaries to marven

Yes, i changed the `jcenter()` repository to `mavenCentral()` and changed the version of the implementation, i commented the lines i changed.

**Comment by alexfernandojs** [comment]
Thanks . really worked I kept only repositories.
Google(),
mavenCentral()
